src/si/dynamic_viscosity.rs

```diff
@@ -0,0 +1,98 @@
+//! Dynamic viscosity (base unit pascal second, kg · m⁻¹ · s⁻¹).
+
+quantity! {
+    /// Dynamic viscosity (base unit pascal second, kg · m⁻¹ · s⁻¹).
+    quantity: DynamicViscosity; "dynamic viscosity";
+    /// Dimension of dynamic viscosity, ML⁻¹T⁻¹ (base unit pascal second, kg · m⁻¹ · s⁻¹).
+    dimension: ISQ<
+        N1,     // length
+        P1,     // mass
+        N1,     // time
+        Z0,     // electric current
+        Z0,     // thermodynamic temperature
+        Z0,     // amount of substance
+        Z0>;    // luminous intensity
+    units {
+        @pascal_second: prefix!(none); "Pa·s",
+            "pascal second", "pascal seconds";
+        @millipascal_second: prefix!(milli); "mPa·s",
+            "millipascal second", "millipascal seconds";
+        @micropascal_second: prefix!(micro); "µPa·s",
+            "micropascal second", "micropascal seconds";
+        // poise = 0.1 Pa · s
+        @poise: 1.0_E-1; "P",
+            "poise", "poises";
+        // centipoise = 1 mPa · s    
+        @centipoise: prefix!(centi) * 1.0_E-1; "cP",
+            "centipoise", "centipoises";  
+        @pound_force_second_per_square_foot: 4.448_222_E0 / 3.048_E-1 / 3.048_E-1; "lbf·s/ft²",
+            "pound-force second per square foot", "pound-force seconds per square foot";          
+        @pound_force_second_per_square_inch: 4.448_222_E0 / 2.54_E-2 / 2.54_E-2; "lbf·s/in²",
+            "pound-force second per square inch", "pound-force seconds per square inch";
+        // Reyn = 1 lbf·s/in²
+        @reyn: 4.448_222_E0 / 2.54_E-2 / 2.54_E-2; "reyn",
+            "reyn", "reyn";
+        @pound_per_foot_second: 4.535_924_E-1 / 3.048_E-1; "lb/(ft·s)",
+            "pound per foot second", "pounds per foot second";      
+        @pound_per_foot_hour: 4.535_924_E-1 / 3.048_E-1 / 3600.0; "lb/(ft·h)",
+            "pound per foot hour", "pounds per foot hour";
+        @slug_per_foot_second: 1.459_390_E1 / 3.048_E-1; "slug/(ft·s)",
+            "slug per foot second", "slugs per foot second";  
+        @gram_per_centimeter_second: prefix!(milli) / prefix!(centi); "g/(cm·s)",
+            "gram per centimeter second", "grams per centimeter second";
+    }
+}
+
+#[cfg(test)]
+mod test {
+    storage_types! {
+        use crate::num::One;
+        use crate::si::quantities::*;
+        use crate::si::dynamic_viscosity as dv;
+        use crate::si::time as t;
+        use crate::si::mass as m;
+        use crate::si::length as l;
+        use crate::si::pressure as p;
+        use crate::tests::Test;
+
+        #[test]
+        fn check_dimension() {
+            let _: DynamicViscosity<V> = Pressure::new::<p::pascal>(V::one())
+                * Time::new::<t::second>(V::one());
+        }
+
+        #[test]
+        fn check_units() {
+            test::<dv::pascal_second, t::second, p::pascal>();
```

Comment:
I'd like to see the generic parameters in formula order: pressure * time = dynamic viscosity.

```suggestion
            test::<p::pascal, t::second, dv::pascal_second>();
```